templates/PyPI-release.yml.tmpl

```diff
@@ -0,0 +1,38 @@
+name: Release on PyPI
+
+defaults:
+  run:
+    shell: bash
+
+on: [workflow_dispatch]
+
+jobs:
+  autorelease:
+    name: Release on TestPyPI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2.3.4
+
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: '{{PYTHON_VERSION}}'
+
+      - name: Check typing, linting, formatting and run unit-tests
+        run: make checks
+
+      - name: Build distribution and wheel
+        run: |
+          python -m pip install build wheel
+          python setup.py sdist bdist_wheel
+
+      - name: Add version to environment vars
+        run: |
+          NAME=$(python setup.py --fullname)
+          echo "NAME=$NAME" >> $GITHUB_ENV
+
+      - name: Publish distribution to PyPI
+        run: |
+          if [[ "${NAME}" =~ .*"+".* ]]; then echo "This is a temporary version: ${NAME}"; exit 1; fi
```

Comment:
Is this temporary version as log says? or is it a typo?

Reply:
Actually, this is the message I'd like to get as command's output when running it on a temporary version (i.e. a version that contains +HASH.dirty* in its tar.gz's name)